lib/progress_bar/enumerable.rb

```diff
@@ -0,0 +1,6 @@
+require_relative '../progress_bar'
+
+# FIXME: should there be a better method?..
+[Enumerable, Array, Hash, Range].each do |mod|
```

Comment:
Yup, you're right of course. I guess we'll just have to leave it up to the user to figure out how to add it to an ActiveRecord::Relation or Mongoid collection...